tests/src/e2e/ibc_tests.rs

```diff
@@ -0,0 +1,1404 @@
+    let msg = MsgChannelCloseConfirm {
+        port_id: port_channel_id_b.port_id.clone(),
+        channel_id: port_channel_id_b.channel_id,
+        proofs,
+        signer: Signer::new("test_b"),
+    };
+    // Update the client state of Chain A on Chain B
+    update_client_with_height(test_a, test_b, client_id_b, height_a)?;
+    // CloseConfirmChannel on Chain B
+    submit_ibc_tx(test_b, msg, ALBERT)?;
+
+    Ok(())
+}
+
+fn get_channel_proofs(
+    test: &Test,
+    client_id: &ClientId,
+    port_channel_id: &PortChannelId,
+    target_height: Height,
+) -> Result<Proofs> {
+    // we need proofs at the height of the previous block
+    let query_height = target_height.decrement().unwrap();
+    let key = channel_key(port_channel_id);
+    let (_, tm_proof) = query_value_with_proof(test, &key, query_height)?;
+    let proof = convert_proof(tm_proof)?;
+
+    let (_, client_state_proof, consensus_proof) =
+        get_client_states(test, client_id, query_height)?;
+
+    Proofs::new(
+        proof,
+        Some(client_state_proof),
+        Some(consensus_proof),
+        None,
+        target_height,
+    )
+    .map_err(|e| eyre!("Creating proofs failed: error {}", e))
+}
+
+// get the client state, the proof of the client state, and the proof of the
+// consensus state
+fn get_client_states(
+    test: &Test,
+    client_id: &ClientId,
+    target_height: Height, // should have been already decremented
+) -> Result<(AnyClientState, CommitmentProofBytes, ConsensusProof)> {
+    let key = client_state_key(client_id);
+    let (value, tm_proof) = query_value_with_proof(test, &key, target_height)?;
+    let client_state = match value {
+        Some(v) => AnyClientState::decode_vec(&v)
+            .map_err(|e| eyre!("Decoding the client state failed: {}", e))?,
+        None => {
+            return Err(eyre!(
+                "The client state doesn't exist: client ID {}",
+                client_id
+            ));
+        }
+    };
+    let client_state_proof = convert_proof(tm_proof)?;
+
+    let height = client_state.latest_height();
+    let key = consensus_state_key(client_id, height);
+    let (_, tm_proof) = query_value_with_proof(test, &key, target_height)?;
+    let proof = convert_proof(tm_proof)?;
+    let consensus_proof = ConsensusProof::new(proof, height)
+        .map_err(|e| eyre!("Creating ConsensusProof failed: error {}", e))?;
+
+    Ok((client_state, client_state_proof, consensus_proof))
+}
+
+fn transfer_token(
+    test_a: &Test,
+    test_b: &Test,
+    client_id_a: &ClientId,
+    client_id_b: &ClientId,
+    port_channel_id_a: &PortChannelId,
+) -> Result<()> {
+    let xan = find_address(test_a, XAN)?;
+    let sender = find_address(test_a, ALBERT)?;
+    let receiver = find_address(test_b, BERTHA)?;
+
+    let token = Some(Coin {
+        denom: xan.to_string(),
+        amount: "100000".to_string(),
+    });
+    let msg = MsgTransfer {
```

Comment:
can we use the cmd from #626 for the transfers?

Reply:
Yes, we can use the `ibc-transfer` cmd for transaction requests with MsgTransfer.